src/Fixer/FunctionNotation/LambdaNotUsedImportFixer.php

```diff
@@ -288,7 +288,7 @@ private function filterArguments(Tokens $tokens, array $arguments): array
             $info = $this->argumentsAnalyzer->getArgumentInfo($tokens, $start, $end);
             $argument = $info->getNameIndex();
 
-            if ($tokens[$tokens->getPrevMeaningfulToken($argument)]->equals('&')) {
+            if ('&' === $tokens[$tokens->getPrevMeaningfulToken($argument)]->getContent()) {
```

Comment:
ref https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5776#discussion_r685784051
i would suggest we check both tokens (but not all 3)

Reply:
can you explain why instead of:
```php
if ('&' === $prevToken->getContent()) {
```
you prefer
```php
if ($prevToken->equals('&') || defined('T_AMPERSAND_FOLLOWED_BY_VAR_OR_VARARG') && $prevToken->isGivenKind(T_AMPERSAND_FOLLOWED_BY_VAR_OR_VARARG)) {
```
?
It is much longer, thus harder to read and comprehend and I cannot see what value it brings.

Reply:
because now `&` can mean different things (in 8.1).
it's like applying rule for all `array` keyword while you want to have it only applied for array creation and not array typehint.

one may do it better than such a long example you provided, but we shouldn't let in more contexts than we want to